app/scripts/background.js

```diff
@@ -1031,7 +1035,9 @@ export function setupController(
   // MetaMask Controller
   //
   controller = new MetamaskController({
-    infuraProjectId: process.env.INFURA_PROJECT_ID,
+    infuraProjectId:
```

Comment:
Can we use the `infuraProjectId` constant from `shared/constants/network.ts` here instead? To avoid duplicating this ternary